src/Libraries/Microsoft.Extensions.AI.Ollama/OllamaChatClient.cs

```diff
@@ -23,6 +23,7 @@ namespace Microsoft.Extensions.AI;
 public sealed class OllamaChatClient : IChatClient
 {
     private static readonly JsonElement _defaultParameterSchema = JsonDocument.Parse("{}").RootElement;
+    private static readonly JsonElement _defaultJsonSchema = JsonDocument.Parse("\"json\"").RootElement;
```

Comment:
What does this string represent? Technically, it's not a valid schema document.

Reply:
It's the default value for the `format` property, i.e., it causes us to produce `format: "json"`. I'll rename this to clarify.